queryzen-client/queryzen/queryzen.py

```diff
@@ -222,6 +252,13 @@ class QueryZen:
     def __init__(self, client: QueryZenClientABC | None = None):
         self._client: QueryZenClientABC = client or QueryZenHttpClient()
 
+    def _normalize_version(self, version):
```

Comment:
issues: missing docstring and type hints, no test
suggestion: change name, it doesn't really normalize but validate
suggestion: probably need to see why we cast the version to string